CONTRIBUTING.md

```diff
@@ -139,6 +145,24 @@ Make sure to signoff your commits before submitting a pull request.
 
 https://docs.pi-hole.net/guides/github/how-to-signoff/
 
+When developing locally, the following commands are useful to run regularly to check unit tests and linting.
+
+| Command  | Description                                        | Is called in the CI? |
+| make unit-test | Runs unit tests only. `make all` will also run this. | yes                  |
+| make check-linter | Checks linter issues only. `make all` will also run this. | yes                  |
+
+## Changing Score Results
+
+As a general rule of thumb, pull requests that change Scorecard score results will need a good reason to do so to get merged. 
```

Comment:
Request that such changes should first be discussed in an issue?

Reply:
https://github.com/ossf/scorecard/pull/3762/commits/9ca2e14078534c8b15ad1f99e76ff58bb67f820e